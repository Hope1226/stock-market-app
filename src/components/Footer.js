import React from 'react';

const Footer = () => (
  <div className="footer">
    <small>
      Â© Hope - 2021. Data is provided by
      {' '}
      <a href="https://site.financialmodelingprep.com/developer/docs">Financial modeling prep</a>
    </small>
  </div>
);

export default Footer;
