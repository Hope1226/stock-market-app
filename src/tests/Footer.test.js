import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Should render the footer page', () => {
  test('and should match the snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON;
    expect(tree).toMatchSnapshot();
  });
});
