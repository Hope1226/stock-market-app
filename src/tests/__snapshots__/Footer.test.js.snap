// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the footer page and should match the snapshot 1`] = `[Function]`;

exports[`Should render the home page and should match the snapshot 1`] = `[Function]`;
